.zuul.yaml

```diff
@@ -0,0 +1,4 @@
+- project:
+  kebechet-auto-gate:
+    queue: "thoth-station/core"
```

Comment:
even the queue could be removed, because we dont need to include PR from other repos...